src/components/TopicMenu/TopicMenu.scss

```diff
@@ -95,6 +95,7 @@
     margin: 2px 0 0 20px;
 
     .rs-layer-tree {
+      margin-top: -1px;
```

Comment:
do we really need this one ?

Reply:
To ensure the pixel precise distance between layer items and topics

Reply:
Why not putting margin-top: 1px to the wkp-layer-tree class instead dof 2px ?

Reply:
maybe remove the wkp-layer-tree element , it seems useless.

Reply:
It works as well indeed